service/src/test/java/bio/terra/workspace/common/BaseUnitTest.java

```diff
@@ -3,6 +3,7 @@
 import org.junit.jupiter.api.Tag;
 import org.springframework.test.context.ActiveProfiles;
 
+/** Base class for (GCP) unit tests: not connected (to GCP) */
```

Comment:
nit: Some of these are GCP functionality unit tests, others are strictly WSM component unit tests. we didn't split those apart when GCP was the only cloud

Reply:
Yep, hence added gcp within brackets. Will update comment to be more descriptive
splitting GCP tests - thought of it during this PR. https://verily.atlassian.net/browse/TERRA-549